frontend/app/src/modules/balances/use-balances-store.ts

```diff
@@ -42,21 +42,60 @@ export const useBalancesStore = defineStore('balances', () => {
     set(manualLiabilities, updatePriceData(get(manualLiabilities), prices));
   };
 
-  const updateBalances = (chain: string, { perAccount }: BlockchainBalances): void => {
+  const updateBlockchainBalances = (chain: string, { perAccount }: BlockchainBalances): void => {
     const data = perAccount[camelCase(chain)] ?? {};
-    set(balances, {
-      ...get(balances),
+    set(blockchainBalances, {
+      ...get(blockchainBalances),
       [chain]: data,
     });
   };
 
+  const removeIgnoredAssets = (ignoredAssets: string[]): void => {
```

Comment:
I did some small changes in the remove method, that should improve performance a bit.

Also added a few tests to make sure that all works fine.